scripts/validation.ipynb

```diff
@@ -840,7 +840,7 @@
     }
    ],
    "source": [
-    "def meta_geog_df(meta_geog):\n",
+    "def meta_geog_df(summary_df, meta_geog):\n",
```

Comment:
modify the methods to not use global variables and instead pass requisite data as arguments.